independent-publisher-connectors/MS Graph Groups and Users/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1151 @@
+                },
+                "onPremisesSyncEnabled": {
+                  "type": "string",
+                  "description": "onPremisesSyncEnabled",
+                  "title": "OnPremises Sync Enabled"
+                },
+                "preferredDataLocation": {
+                  "type": "string",
+                  "description": "preferredDataLocation",
+                  "title": "Preferred Data Location"
+                },
+                "preferredLanguage": {
+                  "type": "string",
+                  "description": "preferredLanguage",
+                  "title": "Preferred Language"
+                },
+                "proxyAddresses": {
+                  "type": "array",
+                  "items": {
+                    "description": "proxyAddresses",
+                    "title": "Proxy Addresses",
+                    "type": "string"
+                  },
+                  "description": "proxyAddresses"
+                },
+                "renewedDateTime": {
+                  "type": "string",
+                  "description": "renewedDateTime",
+                  "title": "Renewed Date Time"
+                },
+                "resourceBehaviorOptions": {
+                  "type": "array",
+                  "items": {
+                    "description": "resourceBehaviorOptions",
+                    "title": "Resource Behavior Options",
+                    "type": "string"
+                  },
+                  "description": "resourceBehaviorOptions"
+                },
+                "resourceProvisioningOptions": {
+                  "type": "array",
+                  "items": {
+                    "description": "resourceProvisioningOptions",
+                    "title": "Resource Provisioning Options",
+                    "type": "string"
+                  },
+                  "description": "resourceProvisioningOptions"
+                },
+                "securityEnabled": {
+                  "type": "boolean",
+                  "description": "securityEnabled",
+                  "title": "Security Enabled",
+                  "enum": [
+                    true,
+                    false
+                  ]
+                },
+                "securityIdentifier": {
+                  "type": "string",
+                  "description": "securityIdentifier",
+                  "title": "Security Identifier"
+                },
+                "theme": {
+                  "type": "string",
+                  "description": "theme",
+                  "title": "Theme"
+                },
+                "visibility": {
+                  "type": "string",
+                  "description": "visibility",
+                  "title": "Visibility"
+                },
+                "onPremisesProvisioningErrors": {
+                  "type": "array",
+                  "items": {
+                    "description": "onPremisesProvisioningErrors",
```

Comment:
@jagjani03 please look through all the descriptions and format them according to the guidelines provided in previous comments.

Reply:
@v-EgorKozhadei description has been updated. Thanks!